inst/extdata/install-packages.R

```diff
@@ -1,26 +1,59 @@
 # install dependencies
 installationDeps <- function(installOption) {
   cat("Installing installation dependencies.\n")
-  
+
   # if user chose local installation, install renv and initialize it
   if (installOption == 1) {
     install.packages("renv")
-    renv::init(bare = TRUE, restart = FALSE)
+    require(renv)
+    init(bare = TRUE, restart = FALSE)
+  }
+
+  # Declare packages
+  packages <- c("cli","remotes","rstudioapi")
+
+  # Loop through each package
+  for (package in packages) {
+
+    # Install package
+    # Note: `installed.packages()` returns a vector of all the installed packages
+    if (!require(package,character.only = T, quietly = T)) {
+      # Install it
+      install.packages(
+        package,
+        dependencies = TRUE
+      )
+    }
+
+    # Load package
+    # Note: `.packages()` returns a vector of all the loaded packages
+    if (!(package %in% .packages())) {
+      # Load it
+      library(
+        package,
+        character.only = TRUE
+      )
+    }
   }
-  install.packages("pkgbuild", quiet = TRUE)
-  install.packages("cli", quiet = TRUE)
-  install.packages("remotes", quiet = TRUE)
-  install.packages("rstudioapi", quiet = TRUE)
 
+  # install.packages("pkgbuild", quiet = TRUE)
```

Comment:
Why the commented stuff?

Reply:
This changed in next PR: https://github.com/esqLABS/esqlabsR/blob/12169d67dcad0a69cbd8e59c7913660985a5cfa9/inst/extdata/install-packages.R#L14

Reply:
No I mean why is the commented stuff still here... it can be removed, right?